php/class-customize-snapshot-manager.php

```diff
@@ -745,6 +746,21 @@ public function enqueue_frontend_scripts() {
 	}
 
 	/**
+	 * Enqueue admin scripts.
+	 *
+	 * These files control the behavior and styling of links to remove settings.
+	 * Published snapshots can't be edited, so these files are not needed on those pages.
+	 */
+	public function enqueue_admin_scripts() {
+		global $post;
+		$handle = 'customize-snapshots-admin';
+		if ( isset( $post->post_type ) && ( Post_Type::SLUG === $post->post_type ) && ( 'publish' !== $post->post_status ) ) {
```

Comment:
Should this be looking at the `$hook` arg passed in? Or should it be checking what is returned by `get_current_screen()`?